src/styles/pages/roar-privacy-tos.scss

```diff
@@ -1,8 +1,23 @@
 .layout.privacy-tos-page {
+  header {
+    position: initial;
+  }
+
   .privacy-tos-section {
-    padding-top: 10rem;
-    h3 {
-      padding: 1rem 0;
+    padding: 0 5rem;
+    text-align: center;
```

Comment:
Add `max-width: 1000px;` to make this easier to read. Change to `padding: 0 25px` to make this consistent with the default padding on other pages. We should have a more standardized way of doing this so that it need not be defined on every page, but that's for another day.

Reply:
changed styles as suggested